api/views/__init__.py

```diff
@@ -343,6 +351,78 @@ def report_webcompat_issue(request):
     return response.Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 
+class FirstForwardedEmailRateThrottle(throttling.UserRateThrottle):
+    rate = settings.FIRST_EMAIL_RATE_LIMIT
+
+
+@decorators.permission_classes([permissions.IsAuthenticated])
+@extend_schema(methods=["POST"], request=FirstForwardedEmailSerializer)
+@decorators.api_view(["POST"])
+@decorators.throttle_classes([FirstForwardedEmailRateThrottle])
+def first_forwarded_email(request):
+    """
+    Requires `free_user_onboarding` flag to be active for the user.
+
+    Send the `first_forwarded_email.html` email to the user via a mask.
+    See [/emails/first_forwarded_email](/emails/first_forwarded_email).
+
+    Note: `mask` value must be a `RelayAddress` that belongs to the authenticated user.
+    A `DomainAddress` will not work.
+    """
+    if not flag_is_active(request, "free_user_onboarding"):
+        # Return Permission Denied error
+        return response.Response(
+            {"detail": "Requires free_user_onboarding waffle flag."}, status=403
+        )
+    serializer = FirstForwardedEmailSerializer(data=request.data)
+    if serializer.is_valid():
```

Comment:
(nit) We could do `if not serializers.is_valid()` and `return response.Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)`, then unindent the line 378 to line 423

Reply:
I love early `return` coding! Nice catch!